runtime/compiler/trj9/env/j9method.cpp

```diff
@@ -4605,7 +4605,7 @@ TR_ResolvedJ9Method::setRecognizedMethodInfo(TR::RecognizedMethod rm)
       TR_OpaqueClassBlock *clazz = fej9()->getClassOfMethod(getPersistentIdentifier());
       J9JITConfig * jitConfig = fej9()->getJ9JITConfig();
       TR::CompilationInfo * compInfo = TR::CompilationInfo::get(jitConfig);
-      TR_PersistentClassInfo *clazzInfo = compInfo->getPersistentInfo()->getPersistentCHTable()->findClassInfo(clazz);
+      TR_PersistentClassInfo *clazzInfo = compInfo->getPersistentInfo()->getPersistentCHTable()->findClassInfoAfterLocking(clazz, fej9());
```

Comment:
How does this code work when CHTable does not exist or someone uses disableCHOpts? I am aware that the new code is no worse than the old one.